internal/utils/configs/manifest.go

```diff
@@ -46,6 +47,12 @@ func (manager *ManifestManager) GetDefaultManifestWithEnv() Manifest {
 func getSkupperImages(components []string, enableSHA bool, runningPods map[string]string) SkupperManifest {
 	var manifest SkupperManifest
 
+	// if Docker is not installed we can not inspect digests from images.
+	_, err := exec.LookPath("docker")
+	if err != nil {
+		enableSHA = false
```

Comment:
I am wondering if it should not print a warning to stderr, in case docker is not available?
Thoughts?

Reply:
done